### DISCUSSION TITLE
Ψηφοφορια για την ΟΣΣ της Κυριακής 12 Μαΐου

### DISCUSSION THREAD

**Discussion by Maurphal** [sdy60/syllabus#141] [Polls]
Καλημέρα και Χρόνια πολλά!

**Comment by epidrome** [comment]
Συγνώμη για την (αδικαιολόγητη) απουσία μου σήμερα, αν βολεύουν οι ίδιες ώρες και δεν υπάρχει κάποια αντίρρηση μπορούμε να κάνουμε αναπλήρωση την επόμενη Κυριακή 19 Μαΐου και φυσικά μπορούμε να κάνουμε μια ακόμη συμπληρωματική πριν την τελική εξέταση
